src/Microsoft.AspNetCore.Mvc.Razor.Precompilation/Internal/CompileOutputs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Precompilation.Internal
+{
+    public class CompileOutputs : IDisposable
+    {
+        public CompileOutputs(string relativePath, bool generatePdbs)
```

Comment:
Reducing the overall file size \ deployment size. We could make it an opt-out feature (as opposed to opt-in) if we care about it one way or the other.